File: Day-02-of-30-Days/Solution.py
# Day 02 of 30 Days 👾

import math
def decide_category(data):
    if data < 18.5:
         return "Underweight"
    elif 18.5 <= data <= 24.9:
         return "Healthy"
    elif 25.0 <= data <= 29.9:
         return "Overweight"
    else:
         return "Obesity"
    
def calculate_bmi(height, weight):
     if height == 0 or weight == 0:
          print("Invalid Input")
          exit(0)
     bmi = weight / math.pow(height/100, 2)
     return {round(bmi*100)/100.0, decide_category(bmi)}

# Example 1
print(calculate_bmi(157.48, 46))

File: Day-02-of-30-Days/Solution.js
// Day 02 of 30 Days 👾

function decidingCategory(data) {
    if (data < 18.5) {
        return "Underweight";
    } else if (data >= 18.5 && data <= 24.9) {
        return "Healthy";
    } else if (data >= 25.0 && data <= 29.9) {
        return "Overweight";
    } else {
        return "Obesity";
    }
}

function calculateBMI(height, weight) {
    if (height == 0 || weight == 0) {
        console.log("Invalid Input");
        process.exit(0);
    }
    let BMI = weight / Math.pow(height / 100, 2);
    return [Math.round(BMI * 100) / 100.0, decidingCategory(BMI)];
}

// Example 1
console.log(calculateBMI(160, 49));